private/scribble-util.rkt

```diff
@@ -0,0 +1,24 @@
+#lang racket/base
+
+(module doc racket/base
+
+  (require racket/contract/base
```

Comment:
Very minor nit: all of Rebellion's modules are organized such that the provided exports are before the imported dependencies (with the exception of `racket/contract/base` because otherwise `contract-out` doesn't work) in order to give a top-level summary of the module as early as possible. Could you do the same thing here?